openlibrary/templates/home/categories.html

```diff
@@ -15,7 +15,7 @@ <h2 class="home-h2"><a href="/subjects">$_('Browse by Subject')</a></h2>
           $ presentable_subject_name = subject_name.replace('_', ' ').title()
           <div class="category-item carousel__item">
             <a class="category-nostyle" data-ol-link-track="CarouselCategories"
-               href="/subjects/$(subject_name)#sort=date_published&ebooks=true">
+               href="/subjects/$(subject_name)?sort=new&has_fulltext=true">
```

Comment:
Filtering to eBooks only on a hardwired basis is a bad idea and confusing to patrons.
```suggestion
               href="/subjects/$(subject_name)?sort=new">
```